app/presenters/activity_presenter.rb

```diff
@@ -282,7 +282,7 @@ def sentence_of_benefitting_countries(country_code_list)
     return nil unless country_code_list.present?
     benefitting_country_names = country_code_list.map { |country_code|
       benefitting_country = BenefittingCountry.find_by_code(country_code)
-      benefitting_country.nil? ? translate("page_content.activity.unknown_country") : benefitting_country.name
+      benefitting_country.nil? ? I18n.t("page_content.activity.unknown_country", code: country_code) : benefitting_country.name
```

Comment:
Nice, it makes so much sense to include full and helpful information in error messages!

Reply:
I probably should have used that `translate` method even though I am not sure why it is there, I might just change that before I merge this!

Reply:
I had to update the `translate` method as it wasn't handling passed in attribute options:

```ruby
def translate(*args)
  I18n.t(*args)
end
```